go-controller/pkg/ovn/controller/services/loadbalancer.go

```diff
@@ -362,8 +362,16 @@ func buildLB(lb *LB) *templateLoadBalancer {
 	// Session affinity
 	// If enabled, then bucket flows by 3-tuple (proto, srcip, dstip) for the specific timeout value
 	// otherwise, use default ovn value
+	selectionFields := []nbdb.LoadBalancerSelectionFields{}
 	if lb.Opts.AffinityTimeOut > 0 {
-		options["affinity_timeout"] = fmt.Sprintf("%d", lb.Opts.AffinityTimeOut)
+		if lb.Opts.AffinityTimeOut != 86000 {
```

Comment:
change this to use https://github.com/kubernetes/kubernetes/blob/35bed806dc7d568d2b98883fed05836fadf9621a/pkg/apis/core/types.go#L4098

Reply:
done